src/devtools.ts

```diff
@@ -0,0 +1,15 @@
+import browser from 'webextension-polyfill'
+import { createEndpointRuntime } from './internal/endpoint-runtime'
+import { createStreamWirings } from './internal/stream'
+import { createPersistentPort } from './internal/persistent-port'
+
+const port = createPersistentPort(`devtools@${browser.devtools.inspectedWindow.tabId}`)
+const endpointRuntime = createEndpointRuntime(
+  'devtools',
+  message => port.postMessage(message),
+)
+
+port.onMessage(endpointRuntime.handleMessage)
+
+export const { sendMessage, onMessage } = endpointRuntime
+export const { openStream, onOpenStreamChannel } = createStreamWirings(endpointRuntime)
```

Comment:
q: what about your idea about removing the stream support from this lib? have you decided to keep it?

Reply:
We'll keep it for `6.0.0`, and decide its fate after